velox/common/base/benchmarks/StringSearchBenchmark.cpp

```diff
@@ -25,7 +25,6 @@
 #include <folly/Benchmark.h>
 #include <folly/Random.h>
 #include <folly/container/Foreach.h>
-#include <folly/portability/GFlags.h>
```

Comment:
gflags is still required to parse commandline args below ...

Reply:
I will add it back. Removing did not help fix the build failure anyway.